third_party/dml/ci/generate_absl_test_summary.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""
+Parses an AbslTest XML log to produce a flattened summary of test results with the following structure:
+{
+    "Counts":
```

Comment:
[minor] Consider naming "Results", just to be consistent with line 20's name (like how you use "Time" and "Errors" inside and outside the "Tests" block).

Reply:
This requires changing a few other scripts as well. I'll keep it that way for now just to stay consistent with the DML scripts as well.

Reply:
Okie dokie.